gatherer_test.go

```diff
@@ -695,6 +697,120 @@ func TestGatherer_SeenObj(t *testing.T) {
 	assert.Equal(t, m, e)
 }
 
+func TestGatherer_NameByPath(t *testing.T) {
```

Comment:
can we add a `t.Parallel()` to this as well as in the subtests below?

Reply:
of course.